cis-kube-benchmark/cis-1.5/ibm/ibm-kube-bench-test-namespace.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Per https://cloud.ibm.com/docs/containers?topic=containers-cis-benchmark#cis-worker-test, we need this to be in the `job-node.yaml` file.